src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.html

```diff
@@ -169,9 +169,9 @@
             <a routerLink="/monitoring">
               <ng-container i18n>Monitoring</ng-container>
               <small *ngIf="prometheusAlertService.activeCriticalAlerts > 0"
-                     class="badge badge-danger ml-1">{{ prometheusAlertService.activeCriticalAlerts }}</small>
+                     class="badge badge-danger ms-1">{{ prometheusAlertService.activeCriticalAlerts }}</small>
               <small *ngIf="prometheusAlertService.activeWarningAlerts > 0"
-                     class="badge badge-warning ml-1">{{ prometheusAlertService.activeWarningAlerts }}</small>
+                     class="badge badge-warning ms-1">{{ prometheusAlertService.activeWarningAlerts }}</small>
```

Comment:
I don't understand why the badges are now almost bubbles, when in [Bootstrap 5.0 they're still square with round corners](https://getbootstrap.com/docs/5.0/components/badge/):
![image](https://user-images.githubusercontent.com/37327689/199107607-1c34cf92-b869-4f62-99b9-154714daa79c.png)

![image](https://user-images.githubusercontent.com/37327689/199107688-be3ae31f-92ac-4294-9260-b87345d66966.png)

Are we happy with that change?

Reply:
I actually overrode the style just to see how it looks. I was going to bring it up in one of the standups but since I was solving the other issues in this PR, I focused on that. Maybe I'll show it in the standup and if it doesn't look I can remove my overrides.